src/tcache.c

```diff
@@ -636,25 +637,23 @@ tcache_arena_reassociate(tsdn_t *tsdn, tcache_slow_t *tcache_slow,
 }
 
 static void
-tcache_max_and_nhbins_init(tcache_t *tcache) {
-	assert(tcache != NULL);
+tcache_slow_settings_init(tcache_slow_t *tcache_slow) {
```

Comment:
nit: maybe tcache_default_settings_init? To make it clear this is init to default which should only happen at creation time.